npm-debug.log

```diff
@@ -0,0 +1,26 @@
+0 info it worked if it ends with ok
```

Comment:
what is this?
if this keeps appearing on windows perhaps putting this file in gitignore will be better